lms/djangoapps/teams/static/teams/js/spec/views/team_profile_spec.js

```diff
@@ -38,13 +43,105 @@ define([
                 )
             );
             AjaxHelpers.respondWithJson(requests, TeamDiscussionSpecHelper.createMockDiscussionResponse());
-            return discussionView;
+            return profileView;
         };
 
-        it('can render itself', function () {
-            var requests = AjaxHelpers.requests(this),
-                view = createTeamProfileView(requests);
-            expect(view.$('.discussion-thread').length).toEqual(3);
+        describe('DiscussionsView', function() {
```

Comment:
This isn't really a test of the DiscussionsView. I'd probably leave this as a top level test called ``'can render itself'`` that verifies that all the basic pieces are in place.